store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/FederatedOperationChainValidator.java

```diff
@@ -47,14 +49,24 @@ public FederatedOperationChainValidator(final ViewValidator viewValidator) {
 
     @Override
     protected Schema getSchema(final Operation operation, final User user, final Store store) {
-        return ((FederatedStore) store).getSchema(operation, user);
+        return (operation instanceof FederatedOperation)
+                ? ((FederatedStore) store).getSchema(getFederatedWrappedSchema().graphIds(((FederatedOperation) operation).getGraphIds()), new Context(user))
+                : ((FederatedStore) store).getSchema(getFederatedWrappedSchema(), new Context(user));
     }
 
     @Override
-    protected void validateViews(final Operation op, final User user, final Store store, final ValidationResult validationResult) {
-        validateAllGraphsIdViews(op, user, store, validationResult, getGraphIdsCSV(op, user, (FederatedStore) store));
+    protected boolean shouldValidate(final Operation op) {
```

Comment:
```suggestion
    protected boolean shouldValidate(final Operation operation) {
```
'operation' is written fully in other functions in this class so I think it would be best to use that throughout this class.
This will also need changing from line 63 and 80

Reply:
:D getSchema was the odd one out in this class.